src/components/breedList/BreedList.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useEffect } from 'react';
+import Breed from '../breed/Breed';
+import { StyledMain, BreedListContainer, StyledInput } from './styles';
+
+export default function BreedList() {
+    const [breeds, setBreeds] = useState<string[]>([]);
+    const [query, setQuery] = useState<string>('');
+
+    function getBreeds() {
+        fetch(`https://dog.ceo/api/breeds/list/all`)
```

Comment:
In the case of the dog api failing for a reason beyond our fault, how do you think we could handle a failing api call ?

Reply:
This is an oversight from me as I should have thought the api might not work sometimes. The correct thing to do is

`if (!response.ok) {

        throw Error(response.statusText);
        
 }`